torch/testing/_internal/common_fsdp.py

```diff
@@ -498,10 +511,12 @@ def _test_identical_outputs(
             )
         else:
             model = ref_ddp_fn(model)
+        if mixed_precision:
+            model = model.half()
```

Comment:
maybe just use auto_cast for ddp tests with mixed precision, same for inputs?

Reply:
We added comparison for single iteration with auto_cast: https://github.com/pytorch/pytorch/pull/75024/files#diff-7b5c66f99161fa6a3d9042e80f8c8cc140a64e43445feede46f55e53154f6c3dR526 in DDP.